sdk/metricsadvisor/Azure.AI.MetricsAdvisor/README.md

```diff
@@ -131,41 +131,59 @@ var adminClient = new MetricsAdvisorAdministrationClient(new Uri(endpoint), new
 
 ### Time Series
 
-A time series is a series of data points indexed in time order. These data points describe the variation of the value of a [metric](#data-feed-metric) over time.
+A time series is a sequence of data points indexed chronologically. These data points describe the variation of the value of a [metric](#data-feed-metric) over time.
 
-Given a metric, the Metrics Advisor service creates one series for every possible [dimension](#data-feed-dimension) value, which means that multiple time series can be monitored for the same metric.
+Given a metric, the Metrics Advisor service creates one series for every possible combination of [dimension](#data-feed-dimension) values, which means that multiple time series can be monitored for the same metric.
+
+For example, suppose the following columns of data are returned by your data source:
+
+|   City   | Category |   Cost  | Revenue |
+| -------- | -------- | ------- | ------- |
+| New York |  Shoes   | 1045.00 | 1345.00 |
+| New York |   Hats   |  670.00 |  502.00 |
+|   Delhi  |  Shoes   |  991.00 | 1009.00 |
+|   Delhi  |   Hats   |  623.00 |  711.00 |
+
+Cost and revenue are the metrics you want the service to monitor, while city and category are the dimensions that characterize those metrics. There are 4 possible dimension combinations in this data:
+
+- City = New York, Category = Shoes
+- City = New York, Category = Hats
+- City = Delhi, Category = Shoes
+- City = Delhi, Category = Hats
+
+For each metric, the service will create 4 time series to monitor data, each one representing one possible dimension combination. Every time a data source ingestion happens, these series will be updated with a new data point, if available in the newly ingested data.
 
 ### Data Point Anomaly
 
-A `DataPointAnomaly`, or simply "anomaly", occurs when a data point in a [time series](#time-series) behaves unexpectedly. It may occur when a data point value is too high or too low, or when its value changes abruptly between close points. You can specify the conditions a data point must satisfy to be considered an anomaly with an `AnomalyDetectionConfiguration`. See the sample [Create an anomaly detection configuration](#create-an-anomaly-detection-configuration) below for more information.
+A `DataPointAnomaly`, or simply "anomaly", occurs when a data point in a [time series](#time-series) behaves unexpectedly. It may occur when a data point value is too high or too low, or when its value changes abruptly between close points. You can specify the conditions a data point must satisfy to be considered an anomaly with an `AnomalyDetectionConfiguration`. After data ingestion happpens, the service applies all existing configurations to the set of new points in search of anomalies. See the sample [Create an anomaly detection configuration](#create-an-anomaly-detection-configuration) below for more information.
```

Comment:
```suggestion
A `DataPointAnomaly`, or simply "anomaly", occurs when a data point in a [time series](#time-series) behaves unexpectedly. It may occur when a data point value is too high or too low, or when its value changes abruptly between close points. You can specify the conditions a data point must satisfy to be considered an anomaly with an `AnomalyDetectionConfiguration`. After data ingestion happens, the service applies all existing configurations to the set of new points in search of anomalies. See the sample [Create an anomaly detection configuration](#create-an-anomaly-detection-configuration) below for more information.
```